### ISSUE TITLE
zmarket issue

### ISSUE THREAD

**Issue by HOGClan** [srcdslab/sm-plugin-zombiereloaded#39]
/cstrike/cfg/sourcemod/zombiereloaded

// This file was auto-generated by SourceMod (v1.11.0.6936)
// ConVars for plugin "zombiereloaded.smx"

// ----------------------------------------------------------------------------
// Ammo Reserve Settings
// ----------------------------------------------------------------------------

ammo_50AE_max 9999       // deagle
ammo_762mm_max 9999      // scout, ak47, g3sg1, aug
ammo_556mm_box_max 9999  // m249
ammo_556mm_max 9999      // galil, sg552, famas, m4a1, sg550
ammo_338mag_max 9999     // awp
ammo_9mm_max 9999        // mp5navy, tmp, glock, elite
ammo_buckshot_max 9999   // m3, xm1014
ammo_45acp_max 9999      // ump45, mac10, usp
ammo_357sig_max 9999     // P228
ammo_57mm_max 9999       // p90, fiveseven

// Attacker receives amount of cash equivalent to the damage that was inflicted.
// -
// Default: "0"
zr_account_cashdmg "1"

// Reset player's cash each spawn.
// -
// Default: "1"
zr_account_cashfill "1"

// Amount of cash to set player's account to. [Dependency: zr_account_cashfill]
// -
// Default: "12000"
zr_account_cashfill_value "16000"

// Play an ambient sound to all players during gameplay.
// -
// Default: "1"
zr_ambientsounds "1"

// Sound file, relative to "sound" folder, to play as ambience.  This file is automatically downloaded to clients. [Dependency: zr_ambientsounds]
// -
// Default: "zr/ambient/zr_ambience.mp3"
zr_ambientsounds_file "zr/ambient/zr_ambience.mp3"

// Length of the ambient sound. [Dependency: zr_ambientsounds]
// -
// Default: "60.0"
zr_ambientsounds_length "60.0"

// Default volume of the ambient sound. [1.0 = Max volume | 0.0001 = Not audible | Dependency: zr_ambientsounds]
// -
// Default: "0.8"
zr_ambientsounds_volume "0"

// Automatically unstick players when stuck within each others' collision hull.
// -
// Default: "1"
zr_antistick "1"

// Allow admins to select admin mode classes. (Not to be confused by admin-only classes!)
// -
// Default: "1"
zr_classes_admin_select "1"

// Time limit to change human class with instant change after spawning. Time is in seconds. Use 0 or negative to disable.
// -
// Default: "20"
zr_classes_change_timelimit "20"

// CS: GO only: Applies an upwards boost if necessary as a workaround for low knock back when standing on the ground. Side effects: Weaker and flying zombies (compensate with lower knock back).
// -
// Default: "1"
zr_classes_csgo_knockback_boost "1"

// (Not implemented!) Admin-only class assigned to admins on connect, if any. ["default" = Default human class | "random" = Random admin-only class]
// -
// Default: "default"
zr_classes_default_admin "default"

// (Incomplete) Admin-mode class assigned to admins on connect. Do not confuse this with admin-only classes. ["random" = Random admin class | "" = Class config default]
// -
// Default: "random"
zr_classes_default_admin_mode "random"

// Human class assigned to players on connect. ["random" = Random human class | "" = Class config default]
// -
// Default: "random"
zr_classes_default_human "random"

// Zombie class assigned to mother zombies. ["motherzombies" = Random mother zombie class | "random" = Random regular zombie class | "disabled" = Don't change class on mother zombies]
// -
// Default: "motherzombies"
zr_classes_default_mother_zombie "motherzombies"

// Zombie class assigned to players on connect. ["random" = Random zombie class | "" = Class config default]
// -
// Default: "random"
zr_classes_default_zombie "random"

// Allow players to select human classes.
// -
// Default: "1"
zr_classes_human_select "1"

// Automatically close class selection menu after selecting a class.
// -
// Default: "1"
zr_classes_menu_autoclose "1"

// Display class selection menu when a player spawn for the first time.
// -
// Default: "0"
zr_classes_menu_join "0"

// Re-display class selection menu every spawn.
// -
// Default: "0"
zr_classes_menu_spawn "0"

// Default class overlay toggle state set on connecting player.
// -
// Default: "0"
zr_classes_overlay_default "0"

// Allow players to toggle class overlay.
// -
// Default: "1"
zr_classes_overlay_toggle "0"

// List of commands to hook that players can use to toggle class overlay. [Dependency: zr_classes_overlay_toggle | Delimiter: ","]
// -
// Default: "nightvision"
zr_classes_overlay_togglecmds "nightvision"

// Player is assigned a random class every spawn. [Override: zr_classes_default_*]
// -
// Default: "0"
zr_classes_random "0"

// Save players' class selections in server cookies and restore when connecting. [Override: zr_classes_default_*]
// -
// Default: "1"
zr_classes_save "1"

// Speed method to use when applying player speed. Do not touch this if you don't know what you're doing! ["lmv" = Lagged movement value | "prop" = Player speed property
// -
// Default: "prop"
zr_classes_speed_method "prop"

// Allow players to select zombie classes.
// -
// Default: "1"
zr_classes_zombie_select "1"

// Path, relative to root sourcemod directory, to downloads file.
// -
// Default: "configs/zr/downloads.txt"
zr_config_path_downloads "configs/zr/downloads.txt"

// Path, relative to root sourcemod directory, to hitgroups config file.
// -
// Default: "configs/zr/hitgroups.txt"
zr_config_path_hitgroups "configs/zr/hitgroups.txt"

// Path, relative to root sourcemod directory, to models config file.
// -
// Default: "configs/zr/models.txt"
zr_config_path_models "configs/zr/models.txt"

// Path, relative to root sourcemod directory, to playerclasses config file.
// -
// Default: "configs/zr/playerclasses.txt"
zr_config_path_playerclasses "configs/zr/playerclasses.txt"

// Path, relative to root sourcemod directory, to weapons config file.
// -
// Default: "configs/zr/weapons.txt"
zr_config_path_weapons "configs/zr/weapons.txt"

// Block blast damage inflicted on self or teammates.
// -
// Default: "1"
zr_damage_block_blast "1"

// Block friendly fire.
// -
// Default: "1"
zr_damage_block_ff "1"

// Read hitgroup damage control from hitgroup config file, disabling this allows all zombie hitgroups to be shot.
// -
// Default: "1"
zr_damage_hitgroups "1"

// Intercept suicide commands only after the first zombie has spawned.
// -
// Default: "1"
zr_damage_suicide_after_infect "1"

// List of client commands to intercept as suicide attempts. [Delimiter: ", "]
// -
// Default: "kill, spectate, jointeam, joinclass, explode"
zr_damage_suicide_cmds "kill, spectate, jointeam, joinclass, explode"

// Intercept suicide commands attempted by humans.
// -
// Default: "0"
zr_damage_suicide_human "0"

// Intercept suicide commands attempted by mother zombies.
// -
// Default: "1"
zr_damage_suicide_mzombie "1"

// Intercept suicide commands attempted by zombies.
// -
// Default: "0"
zr_damage_suicide_zombie "1"

// Enable hitgroups module, disabling this will disable hitgroup-related features. (hitgroup knockback multipliers, hitgroup damage control)
// -
// Default: "1"
zr_hitgroups "1"

// Prevent a player from being chosen as mother zombie two rounds in a row.
// -
// Default: "1"
zr_infect_consecutive_block "1"

// Emit an energy splash from player on infection.
// -
// Default: "1"
zr_infect_esplash "1"

// Weapon icon used when the death event is sent for a player that gets infected.
// -
// Default: "zombie_claws_of_death"
zr_infect_event_weapon "zombie_claws_of_death"

// Disabling this will disable the fireball, smoke cloud, and sparks in a more efficient way.
// -
// Default: "1"
zr_infect_explosion "1"

// Spawn a fireball effect around player on infection. [Dependency: zr_infect_explosion]
// -
// Default: "1"
zr_infect_fireball "1"

// Time in seconds during which knife can not be used after becoming a zombie.
// -
// Default: "1.0"
zr_infect_knife_cooldown "1.0"

// The maximum allowed distance between a client and an attacker for a successful infection. [0.0 = Disabled]
// -
// Default: "80.0"
zr_infect_max_distance "80.0"

// Counts down to the first infection of the round.  Countdown is printed in the middle of the client's screen.
// -
// Default: "1"
zr_infect_mzombie_countdown "1"

// Maximum number of mother zombies. Range mode only, cannot be zero.
// -
// Default: "3"
zr_infect_mzombie_max "3"

// Minimum number of mother zombies. Range mode only, cannot be zero.
// -
// Default: "1"
zr_infect_mzombie_min "1"

// Mother zombie infection mode. ['dynamic' = every n-th zombie (ratio) | 'absolute' = n zombies (ratio) | 'range' = min/max]
// -
// Default: "dynamic"
zr_infect_mzombie_mode "dynamic"

// Dynamic mode: Infection ratio. Every n-th player is infected. | Absolute mode: Number of zombies to infect (positive ratio), or number of humans to keep (negative ratio).
// -
// Default: "7"
zr_infect_mzombie_ratio "7"

// Teleport mother zombies back to spawn on infect.
// -
// Default: "0"
zr_infect_mzombie_respawn "1"

// Use randomized round-robin (with SteamID cache) for mother zombie infection.
// -
// Default: "1"
zr_infect_round_robin "1"

// Roll the dice for players new to the random round-robin SteamID cache to make it fair for late joiners.
// -
// Default: "1"
zr_infect_round_robin_rtd "1"

// Shake player's view on infect.
// -
// Default: "1"
zr_infect_shake "1"

// Amplitude of shaking effect. [Dependency: zr_infect_shake]
// -
// Default: "15.0"
zr_infect_shake_amp "15.0"

// Duration of shaking effect. [Dependency: zr_infect_shake]
// -
// Default: "5.0"
zr_infect_shake_duration "5.0"

// Frequency of shaking effect. [Dependency: zr_infect_shake]
// -
// Default: "1.0"
zr_infect_shake_frequency "1.0"

// Spawn a smoke cloud effect around player on infection. [Dependency: zr_infect_explosion]
// -
// Default: "1"
zr_infect_smoke "1"

// Emit sparks from player on infection. [Dependency: zr_infect_explosion]
// -
// Default: "1"
zr_infect_sparks "1"

// Maximum time from the start of the round until picking the mother zombie(s).
// -
// Default: "50.0"
zr_infect_spawntime_max "20.0"

// Minimum time from the start of the round until picking the mother zombie(s).
// -
// Default: "30.0"
zr_infect_spawntime_min "10.0"

// Teleports on infect to zombie.
// -
// Default: "0"
zr_infect_teleport "0"

// Force player to drop all weapons on infect, disabling this will strip weapons instead.
// -
// Default: "1"
zr_infect_weapons_drop "1"

// The maximum horizontal velocity a player can achieve before bunnyhop protection kicks in. [Dependency: zr_jumpboost_bhop_protect]
// -
// Default: "300"
zr_jumpboost_bhop_max "300"

// Prevent players from using forward jump boost multipliers to bunny hop.
// -
// Default: "1"
zr_jumpboost_bhop_protect "1"

// Set maximum velocity zombies can get from knockback ['0' = Off].
// -
// Default: "0"
zr_knockback_maxvel "0"

// Enable logging of events in the plugin. Fatal errors are always logged.
// -
// Default: "1"
zr_log "1"

// Always log error messages no matter what logging flags or modules filters that are enabled.
// -
// Default: "1"
zr_log_error_override "1"

// A bit field that specify what event types to log. See logging section (3.3) in manual for details.
// -
// Default: "2"
zr_log_flags "2"

// Don't log events triggered by console commands that are executed by the console itself, like commands in configs. Enable this command to avoid spamming logs with events like weapon restrictions.
// -
// Default: "1"
zr_log_ignore_console "1"

// Enable module filtering. Only events from listed modules will be logged.
// -
// Default: "0"
zr_log_module_filter "0"

// Print log events to admin chat in addition to the log file.
// -
// Default: "0"
zr_log_print_admins "0"

// Print log events to public chat in addition to the log file.
// -
// Default: "0"
zr_log_print_chat "0"

// Minimum water-saturation before flame is extinguished. ['0' = Disabled | '1' = Feet | '2' = Waist | '3' = Full submersion]
// -
// Default: "0"
zr_napalm_douse "0"

// Ignite grenade in mid-air after player throws it. [Dependency: Human Attribute 'has_napalm']
// -
// Default: "1"
zr_napalm_ignite "1"

// The burn-time is reset when being naded multiple times. [0: Original burn-time is used.]
// -
// Default: "1"
zr_napalm_time_reset "1"

// The amount of damage that's needed to apply full burn duration.
// -
// Default: "75"
zr_napalm_time_scale "75"

// Minimum DirectX level allowed for overlays (mat_dxlevel).
// -
// Default: "80"
zr_overlays_min_dxlevel "80"

// How often to update overlays on players. [0.0 = Disabled]
// -
// Default: "1.0"
zr_overlays_update_time "1.0"

// Use group authentication instead of flags to access admin features. Generic admin flag is still required on some features.
// -
// Default: "0"
zr_permissions_use_groups "0"

// Respawn players after death. [Recommended: (Enable) zr_zspawn*]
// -
// Default: "0"
zr_respawn "0"

// Time after death to delay player respawn. [Dependency: zr_respawn]
// -
// Default: "1"
zr_respawn_delay "1"

// Respawn player as a zombie. [Dependency: zr_respawn]
// -
// Default: "1"
zr_respawn_team_zombie "1"

// Respawn player as a zombie if player was a zombie and killed by world damage. [Override: zr_respawn_team_zombie]
// -
// Default: "1"
zr_respawn_team_zombie_world "1"

// Balances the team every time the round ends. Disable this if you use something else to balance teams.
// -
// Default: "1"
zr_roundend_balance_teams "1"

// Show specified overlay to players depending on winner when the round ends.
// -
// Default: "1"
zr_roundend_overlay "1"

// Overlay, relative to "materials" folder, to display when zombies win the round. [Dependency: zr_roundend_overlay]
// Default: "overlays/zr/zombies_win"
zr_roundend_overlays_zombie "overlays/zr/zg_zombies_win"

// Overlay, relative to "materials" folder, to display when humans win the round. [Dependency: zr_roundend_overlay]
// Default: "overlays/zr/humans_win"
zr_roundend_overlays_human "overlays/zr/zg_humans_win"

// Should zombies win if roundtime runs out.
// -
// Default: "1"
zr_roundend_zombies_win "1"

// Number of sound commands allowed within the time span, or total limit if time span is disabled. ['0' = Disable sound command limit]
// -
// Default: "4"
zr_seffects_command_limit "4"

// Time span for sound command limiter (in seconds). ['0' = Disable time span check | positive number = Time span]
// -
// Default: "10"
zr_seffects_command_timespan "10"

// Max countdown sound file index.
// -
// Default: "10"
zr_seffects_countdown_max "10"

// Emit a death sound when a zombie dies.
// -
// Default: "1"
zr_seffects_death "1"

// Max death sound file index.
// -
// Default: "3"
zr_seffects_die_max "3"

// The probability that a groan sound will be emitted from a zombie when shot. ['100' = 1% chance | '50' = 2% chance | '1' = 100% chance]
// -
// Default: "5"
zr_seffects_groan "5"

// Time between emission of a moan sound from a zombie.
// -
// Default: "30.0"
zr_seffects_moan "30.0"

// Max moaning sound file index.
// -
// Default: "14"
zr_seffects_moan_max "2"

// Max groan sound file index.
// -
// Default: "6"
zr_seffects_pain_max "6"

// Sound, relative to "sound" folder, to play from player when mother zombie countdown starts.
// -
// Default: "zr/countdown/%i.mp3"
zr_seffects_path_countdown "zr/countdown/%i.mp3"

// Sound, relative to "sound" folder, to play from player when killed.
// -
// Default: "zr/die/zombie_die%d.mp3"
zr_seffects_path_die "zr/die/zombie_die%d.mp3"

// Sound, relative to "sound" folder, to play from player when moaning.
// -
// Default: "zr/moan/zombie_voice_idle%d.mp3"
zr_seffects_path_moan "zr/moan/zombie_voice_idle%d.mp3"

// Sound, relative to "sound" folder, to play from player when hurt.
// -
// Default: "zr/pain/zombie_pain%d.mp3"
zr_seffects_path_pain "zr/pain/zombie_pain%d.mp3"

// Sound, relative to "sound" folder, to play from zombie when infects a human.
// -
// Default: "zr/scream/zombie_scream%d.mp3"
zr_seffects_path_scream "zr/scream/zombie_scream%d.mp3"

// Max scream sound file index.
// -
// Default: "1"
zr_seffects_scream_max "1"

// Default volume of the seffects sounds. [1.0 = Max volume | 0.0001 = Not audible]
// -
// Default: "0.8"
zr_seffects_volume "0.8"

// Player will be protected from infection when spawning into the game late.
// -
// Default: "1"
zr_spawnprotect "1"

// Alpha of the player during protection. ['255' = Fully visible | '0' = Completely invisible | Dependency: zr_spawnprotect]
// -
// Default: "0"
zr_spawnprotect_alpha "0"

// Speed of the player during protection. See the manual for more information. [Dependency: zr_spawnprotect]
// -
// Default: "250.0"
zr_spawnprotect_speed "250.0"

// Amount of time to protect player. [Dependency: zr_spawnprotect]
// -
// Default: "10"
zr_spawnprotect_time "10"

// (UNSUPPORTED) Enable fog rendering on the map.
// -
// Default: "0"
zr_veffects_fog "0"

// (UNSUPPORTED) Density (thickness) of the fog. [Dependency: zr_veffects_fog]
// -
// Default: "0.8"
zr_veffects_fog_density "0.8"

// (UNSUPPORTED) Distance from player to stop rendering fog. [Dependency: zr_veffects_fog]
// -
// Default: "400"
zr_veffects_fog_enddist "400"

// (UNSUPPORTED) Vertical clipping plane.
// -
// Default: "2000"
zr_veffects_fog_farz "2000"

// (UNSUPPORTED) If fog exists already on the map, then replace with new modified fog. [Dependency: zr_veffects_fog]
// -
// Default: "0"
zr_veffects_fog_override "0"

// (UNSUPPORTED) Primary color of the fog. [Dependency: zr_veffects_fog]
// -
// Default: "255 255 255"
zr_veffects_fog_pcolor "255 255 255"

// (UNSUPPORTED) Secondary color of the fog. [Dependency: zr_veffects_fog]
// -
// Default: "255 255 255"
zr_veffects_fog_scolor "255 255 255"

// (UNSUPPORTED) Distance from player to start rendering foremost fog. [Dependency: zr_veffects_fog]
// -
// Default: "0"
zr_veffects_fog_startdist "0"

// Change lightstyle (brightness) of the map.
// -
// Default: "0"
zr_veffects_lightstyle "0"

// Lightstyle value. ['a' = Darkest | 'z' = Brightest | Dependency: zr_veffects_lightstyle]
// -
// Default: "b"
zr_veffects_lightstyle_value "b"

// Time to wait before removing the ragdoll. [Dependency: zr_veffects_ragdoll_remove]
// -
// Default: "0.5"
zr_veffects_ragdoll_delay "0.5"

// The ragdoll removal effect. ['-2' = Effectless removal | '-1' = Random effect | '0' = Energy dissolve | '1' = Heavy electrical dissolve | '2' = Light electrical dissolve | '3' = Core dissolve | Dependency: zr_veffects_ragdoll_remove]
// -
// Default: "-1"
zr_veffects_ragdoll_dissolve "-1"

// Remove players' ragdolls from the game after a delay.
// -
// Default: "1"
zr_veffects_ragdoll_remove "1"

// Change map skybox.
// -
// Default: "0"
zr_veffects_sky "0"

// Skybox file, relative to "materials/skybox" folder, to change map skybox to.  This file is automatically downloaded to clients. [Dependency: zr_veffects_sky]
// -
// Default: "sky_borealis01up.vmt"
zr_veffects_sky_path "sky_borealis01up.vmt"

// Disable sun rendering on map.
// -
// Default: "0"
zr_veffects_sun_disable "0"

// (Incomplete) Modify sv_alltalk to obey zombie/human teams instead of t/ct.
// -
// Default: "0"
zr_voice "0"

// Enables volumetric features.
// -
// Default: "1"
zr_vol "1"

// How often to check for delayed events, in seconds. Use lower values for more precise delays.
// -
// Default: "0.5"
zr_vol_trigger_interval "0.5"

// How often to update player positions and trigger events, in seconds.
// -
// Default: "0.5"
zr_vol_update_interval "0.5"

// Enable weapons module, disabling this will disable any weapons-related features. (weapon restrictions, weapon knockback multipliers, etc)
// -
// Default: "1"
zr_weapons "1"

// Give a new knife weapon when player is infected.
// -
// Default: "1"
zr_weapons_infect_removeknife "1"

// Enable weapon restriction module, disabling this will disable weapon restriction commands.
// -
// Default: "1"
zr_weapons_restrict "1"

// Restricts zombies from picking up weapons after the round has ended but before the next round has begun.
// -
// Default: "1"
zr_weapons_restrict_endequip "1"

// Allow player to buy from a list of weapons in the weapons config.
// -
// Default: "1"
zr_weapons_zmarket "1"

// Requires player to be inside a buyzone to use ZMarket. [Dependency: zr_weapons_zmarket]
// -
// Default: "1"
zr_weapons_zmarket_buyzone "0"

// Purchases are free before the first zombie spawns. [Dependency: zr_weapons_zmarket]
// -
// Default: "1"
zr_weapons_zmarket_freespawn "1"

// Dont allow users to rebuy weapons to refill their clip. [Dependency: zr_weapons_zmarket]
// -
// Default: "1"
zr_weapons_zmarket_norefill "1"

// Allow players to rebuy their previous weapons. [Dependency: zr_weapons_zmarket]
// -
// Default: "1"
zr_weapons_zmarket_rebuy "1"

// Automatically equip default or loadout weapons. [Dependency: zr_weapons_zmarket&zr_weapons_zmarket_rebuy]
// -
// Default: "1"
zr_weapons_zmarket_rebuy_auto "1"

// Default primary weapon. [Dependency: zr_weapons_zmarket&zr_weapons_zmarket_rebuy]
// -
// Default: "P90"
zr_weapons_zmarket_rebuy_primary "P90"

// Default secondary weapon. [Dependency: zr_weapons_zmarket&zr_weapons_zmarket_rebuy]
// -
// Default: "Elite"
zr_weapons_zmarket_rebuy_secondary "Elite"

// Allow player to toggle real HP display as a zombie.
// -
// Default: "1"
zr_zhp "1"

// Default ZHP toggle state set on connecting player. [Dependency: zr_zhp]
// -
// Default: "1"
zr_zhp_default "1"

// Allow players to spawn into the game late.
// -
// Default: "1"
zr_zspawn "1"

// Should zspawn spawn joining players?
// -
// Default: "1"
zr_zspawn_auto "1"

// Block players disconnecting and rejoing the game using zspawn.
// -
// Default: "1"
zr_zspawn_block_rejoin "1"

// Override spawn team when spawning by means of ZSpawn.
// -
// Default: "1"
zr_zspawn_team_override "1"

// Spawn player on zombie team when spawning by means of ZSpawn. [Dependency: zr_zspawn_team_override | Override: zr_respawn_team_zombie]
// -
// Default: "0"
zr_zspawn_team_zombie "0"

// Put a time limit on the use of ZSpawn.
// -
// Default: "1"
zr_zspawn_timelimit "1"

// Time from the start of the round to allow ZSpawn. [Dependency: zr_zspawn_timelimit]
// -
// Default: "120.0"
zr_zspawn_timelimit_time "120.0"

// Spawn player on the zombie team AFTER the timelimit is up. ['-1' = Block ZSpawn | '0' = Spawn as human | '1' = Spawn as zombie | Dependency: zr_zspawn_timelimit]
// -
// Default: "1"
zr_zspawn_timelimit_zombie "1"

// Automatically cancel ZTele if player moves out of a set boundary. [Dependency: zr_ztele_(zombie)/(human)[_(before)/(after)]]
// -
// Default: "1"
zr_ztele_autocancel "1"

// Maximum distance, in feet, player is allowed to travel before teleport is cancelled. [Dependency: zr_ztele_autocancel]
// -
// Default: "20"
zr_ztele_autocancel_distance "20"

// Time between using ZTele command and teleportation for humans. [Dependency: zr_ztele_human_(before)/(after)]
// -
// Default: "3.0"
zr_ztele_delay_human "3.0"

// Time between using ZTele command and teleportation for zombies. [Dependency: zr_ztele_zombie]
// -
// Default: "3.0"
zr_ztele_delay_zombie "3.0"

// Teleport the new chosen client infected for replace the last zombie disconnected.
// -
// Default: "1"
zr_ztele_disconnect "1"

// Allow humans to use ZTele after the mother zombie has spawned.
// -
// Default: "1"
zr_ztele_human_after "1"

// Allow humans to use ZTele before the mother zombie has spawned.
// -
// Default: "1"
zr_ztele_human_before "1"

// Max number of times a human is allowed to use ZTele per round. [Dependency: zr_ztele_human_(before)/(after)]
// -
// Default: "1"
zr_ztele_max_human "3"

// Max number of times a zombie is allowed to use ZTele per round. [Dependency: zr_ztele_zombie]
// -
// Default: "3"
zr_ztele_max_zombie "5"

// Teleport player to a random spawn point instead of the players initial one, when using ztele.
// -
// Default: "0"
zr_ztele_random_spawn "0"

// Allow zombies to use ZTele.
// -
// Default: "1"
zr_ztele_zombie "1"

error logs

L 06/08/2024 - 18:53:03: SourceMod error session started
L 06/08/2024 - 18:53:03: Info (map "ze_otakuroom_v5_6fs") (file "/home/container/cstrike/addons/sourcemod/logs/errors_20240608.log")
L 06/08/2024 - 18:53:03: [SM] Exception reported: Invalid Handle 0 (error: 4)
L 06/08/2024 - 18:53:03: [SM] Blaming: zombiereloaded.smx
L 06/08/2024 - 18:53:03: [SM] Call stack trace:
L 06/08/2024 - 18:53:03: [SM]   [0] GetArraySize
L 06/08/2024 - 18:53:03: [SM]   [1] Line 452, restrict::RestrictTypeToIndex
L 06/08/2024 - 18:53:03: [SM]   [2] Line 292, restrict::RestrictWeapon
L 06/08/2024 - 18:53:03: [SM]   [3] Line 719, restrict::RestrictCommand
L 06/08/2024 - 18:53:03: [zombiereloaded.smx] [Sound Effects] [Config Validation] Ambient sound volume specified in "zr_ambientsounds_volume" is either muted or invalid.
L 06/08/2024 - 18:53:07: [CRASH] Accelerator uploaded crash dump: Crash ID: RMV4-KL7K-TRJZ
L 06/08/2024 - 18:54:22: [SM] Exception reported: Invalid index -1 (count: 1)
L 06/08/2024 - 18:54:22: [SM] Blaming: zombiereloaded.smx
L 06/08/2024 - 18:54:22: [SM] Call stack trace:
L 06/08/2024 - 18:54:22: [SM]   [0] GetArrayCell
L 06/08/2024 - 18:54:22: [SM]   [1] Line 557, restrict::RestrictIsWeaponRestricted
L 06/08/2024 - 18:54:22: [SM]   [2] Line 1360, zmarket::ZMarketRebuy
L 06/08/2024 - 18:54:22: [SM]   [3] Line 280, zmarket::ZMarketCheckRebuy
L 06/08/2024 - 18:54:22: [SM]   [4] Line 257, zmarket::ZMarketOnClientSpawnPost
L 06/08/2024 - 18:54:22: [SM]   [5] Line 382, zr/weapons/weapons.inc::WeaponsOnClientSpawnPost
L 06/08/2024 - 18:54:22: [SM]   [6] Line 213, zr/event::EventPlayerSpawnPost
L 06/08/2024 - 18:54:28: [SM] Exception reported: Invalid index -1 (count: 1)
L 06/08/2024 - 18:54:28: [SM] Blaming: zombiereloaded.smx
L 06/08/2024 - 18:54:28: [SM] Call stack trace:
L 06/08/2024 - 18:54:28: [SM]   [0] GetArrayCell
L 06/08/2024 - 18:54:28: [SM]   [1] Line 557, restrict::RestrictIsWeaponRestricted
L 06/08/2024 - 18:54:28: [SM]   [2] Line 1360, zmarket::ZMarketRebuy
L 06/08/2024 - 18:54:28: [SM]   [3] Line 280, zmarket::ZMarketCheckRebuy
L 06/08/2024 - 18:54:28: [SM]   [4] Line 257, zmarket::ZMarketOnClientSpawnPost
L 06/08/2024 - 18:54:28: [SM]   [5] Line 382, zr/weapons/weapons.inc::WeaponsOnClientSpawnPost
L 06/08/2024 - 18:54:28: [SM]   [6] Line 213, zr/event::EventPlayerSpawnPost

also, the rebuy and zmarket does not work after restarting the server

**Comment by maxime1907** [comment]
did you install all the required files correctly ?

**Comment by vdgamee98** [comment]
yes, I deleted and reinstalled it correctly several times, but this error does not disappear

**Comment by Rushaway** [comment]
duplicate of https://github.com/srcdslab/sm-plugin-zombiereloaded/issues/19

**Comment by Rushaway** [comment]
Could you sent the FixMemoryLeak config plugin you was running ? I will try to repro during the weekend.
